sdk/compute/Microsoft.Azure.Management.Compute/src/Microsoft.Azure.Management.Compute.csproj

```diff
@@ -9,7 +9,7 @@
       Provides developers with libraries for the updated compute platform under Azure Resource manager to deploy virtual machine, virtual machine extensions and availability set management capabilities. Launch, restart, scale, capture and manage VMs, VM Extensions and more. Note: This client library is for Virtual Machines under Azure Resource Manager.
       Development of this library has shifted focus to the Azure Unified SDK. The future development will be focused on "Azure.ResourceManager.Compute" (https://www.nuget.org/packages/Azure.ResourceManager.Compute/). Please see the package changelog for more information.
     </Description>
-    <Version>50.0.0</Version>
+    <Version>51.0.0</Version>
     <AssemblyName>Microsoft.Azure.Management.Compute</AssemblyName>
     <PackageTags>management;virtual machine;compute;</PackageTags>
     <PackageReleaseNotes>
```

Comment:
Hi @haagha , Can you update both the PackageReleaseNotes here and the ApiTag in [AzSdk.RP.props](https://github.com/Azure/azure-sdk-for-net/blob/feature/cplat-2021-08-01/sdk/compute/Microsoft.Azure.Management.Compute/AzSdk.RP.props#L4)?

Reply:
Updated, thanks.